### PR TITLE
add "InsecureSkipVerify" option on client configuration

### PR DESCRIPTION
**Pull request by toorop**
this change adds the ability to easily use client via https with a server having self-signed certificate.

### CODE REVIEW

File: client/influxdb.go [hunk 1]
```diff
@@ -73,18 +74,22 @@ func ParseConnectionString(path string, ssl bool) (url.URL, error) {
 // Username/Password are optional.  They will be passed via basic auth if provided.
 // UserAgent: If not provided, will default "InfluxDBClient",
 // Timeout: If not provided, will default to 0 (no timeout)
+// InsecureSkipVerify: controls whether a client verifies the
```
Comment (otoolep) [inline client/influxdb.go]: I know naming is hard. I think a terser option would be best. How about:

`VerifySkip`

Simple as that?

Comment (otoolep) [inline client/influxdb.go]: Oh, I see you're copying the config option name from the TLS client. In that case, the name must be good. :-)

**Comment by otoolep** [comment]
Thanks @toorop -- useful feature.

**Comment by runner-mei** [comment]
why not merge it? it is useful.

**Comment by otoolep** [comment]
@corylanou -- can I get a review? I will rebase and merge.

I have confirmed the CLA has been signed.

**Comment by corylanou** [comment]
Looks good.  Is there a way to test it?  +1

**Comment by otoolep** [comment]
@corylanou -- in theory we would have to spin up a secured server with self-signed cert. Do we do anything testing like that now in our tests? Spin up servers to test client software?

**Comment by corylanou** [comment]
I think we can just spin up a test HTTP server and mock out a the ping endpoint that the cli connects to.  Something similar to https://github.com/influxdb/influxdb/blob/master/cmd/influx/main_test.go#L158

**Comment by otoolep** [comment]
Agreed, I'd like to see that test added (spin up a TLS HTTP server, and ensure connection is succesfull) before we merge.

**Comment by otoolep** [comment]
This has already been added to the client. Please let me know if I am incorrect.

**Comment by toorop** [comment]
Happy to see that you have appreciate my idea & code for your V2 client...

**Pull request closed without merging**